build.sbt

```diff
@@ -1884,7 +1885,8 @@ lazy val `std-table` = project
       "com.univocity"       % "univocity-parsers" % "2.9.1",
       "org.apache.poi"      % "poi-ooxml"         % "5.2.2",
       "org.apache.xmlbeans" % "xmlbeans"          % "5.1.0",
-      "org.graalvm.truffle" % "truffle-api"       % graalVersion % "provided"
+      "org.graalvm.truffle" % "truffle-api"       % graalVersion % "provided",
+      "org.netbeans.api"    % "org-openide-util-lookup" % netbeansApiVersion
```

Comment:
I'm not sure this is needed, I would even advise against this.

`std-table` already depends on `std-base`, so we guarantee it can access all libraries required by `std-table`. I'm not sure if adding it here adds an additional JAR to the distribution? Probably not, I see that ICU is there too.

I think we should remove ICU too as it's also already in Base. Unless it messes up with IntelliJ's classpath detection - then let's keep it.